TESTS/mbed_hal_fpga_ci_test_shield/uart/main.cpp

```diff
@@ -270,7 +270,7 @@ static void uart_test_common(int baudrate, int data_bits, SerialParity parity, i
     tester.reset();
 }
 
-void test_init_free(PinName tx, PinName rx, PinName cts = NC, PinName rts = NC)
+void fpga_uart_init_free_test(PinName tx, PinName rx, PinName cts = NC, PinName rts = NC)
```

Comment:
```suggestion
void fpga_uart_init_free_test(PinName tx, PinName rx, PinName cts, PinName rts)
```
Defaults moved to the header file.

Reply:
This comment is still valid. Defaults should always go in the header file.